docs/assets/auth/oauth-popup-flow.svg

```diff
@@ -1,56 +1 @@
="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="198" x="19" y="856.1934">a reload or an expiring session.</text><polygon fill="#383838" points="827.5,901.5566,837.5,905.5566,827.5,909.5566,831.5,905.5566" style="stroke: #383838; stroke-width: 1.0;"/><line style="stroke: #383838; stroke-width: 1.0;" x1="120.5" x2="833.5" y1="905.5566" y2="905.5566"/><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="185" x="127.5" y="885.5039">GET /auth/&lt;provider&gt;/token</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="194" x="127.5" y="900.8145">Refresh Token cookie included</text><polygon fill="#383838" points="1095.5,961.4883,1105.5,965.4883,1095.5,969.4883,1099.5,965.4883" style="stroke: #383838; stroke-width: 1.0;"/><line style="stroke: #383838; stroke-width: 1.0;" x1="839.5" x2="1101.5" y1="965.4883" y2="965.4883"/><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="90" x="846.5" y="930.125">Refresh Token</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="55" x="846.5" y="945.4355">Client ID</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="79" x="846.5" y="960.7461">Client Secret</text><polygon fill="#383838" points="850.5,1036.7305,840.5,1040.7305,850.5,1044.7305,846.5,1040.7305" style="stroke: #383838; stroke-width: 1.0;"/><line style="stroke: #383838; stroke-width: 1.0;" x1="844.5" x2="1106.5" y1="1040.7305" y2="1040.7305"/><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="87" x="856.5" y="990.0566">Access Token</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="65" x="856.5" y="1005.3672">(ID Token)</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="37" x="856.5" y="1020.6777">Scope</text><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="74" x="856.5" y="1035.9883">Expire Time</text><polygon fill="#383838" points="131.5,1066.041,121.5,1070.041,131.5,1074.041,127.5,1070.041" style="stroke: #383838; stroke-width: 1.0;"/><line style="stroke: #383838; stroke-width: 1.0;" x1="125.5" x2="838.5" y1="1070.041" y2="1070.041"/><text fill="#000000" font-family="Segoe UI, Helvetica, Arial, sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="102" x="137.5" y="1065.2988">Tokens and info</text><!--MD5=[c025fb26ac34956dc15457f201ee7a22]
```

Comment:
So, these were written by hand and will be maintained by hand, so compressing them in-place is probably not desired. Maybe there's some flow we could have where the source is in one place and a built optimized version is in the assets dir?

Reply:
Probably no need to duplicate the image / set up a flow.  Looks like there's a nice ignore image configuration available: https://imgbot.net/docs/#ignoring-files

Reply:
(Proposed in change below)